app_spec/test/files/test.js

```diff
@@ -7,7 +7,7 @@ module.exports = scenario => {
     t.notEqual(alice.info('app').agentAddress, bob.info('app').agentAddress)
   })
 
-  scenario('send ping', async (s, t) => {
+  scenario.only('send ping', async (s, t) => {
```

Comment:
This needs to be removed before merging:
```suggestion
  scenario('send ping', async (s, t) => {
```